code/modules/mining/equipment/mining_tools.dm

```diff
@@ -63,6 +63,16 @@
 	desc = "A pickaxe with a diamond pick head. Extremely robust at cracking rock walls and digging up dirt."
 	force = 19
 
+/obj/item/pickaxe/cyborg
+	name = "cyborg pickaxe"
+	desc = "An integrated pickaxe."
+
+/obj/item/pickaxe/cyborg/Initialize()
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, CYBORG_ITEM_TRAIT)
+	if(CONFIG_GET(flag/disable_standardborg)) //rip
+		desc = "I used to rule the world..."
+
```

Comment:
none of this needs to exist, there is nothing different about this pickaxe at all

Reply:
why does the mining cyborg's drill get its own subtype, then?

also, I think my joke is funny.

Reply:
you should feel free to remove it too